tools/scoring/tests/dimensions/entity_connection_identification_test.py

```diff
@@ -12,41 +12,187 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Test for configuration file scoring tool
-"entity connection identification" dimension
-(entity_connection_identification.py)."""
+"entity connection identification" dimension."""
 
 from absl.testing import absltest
 
 from score.dimensions.entity_connection_identification import EntityConnectionIdentification
-from score.constants import FileTypes
+from score.constants import FileTypes, DimensionCategories
+
+from tests.helper import TestHelper
 
 PROPOSED, SOLUTION = FileTypes
 
 
 class EntityConnectionIdentificationTest(absltest.TestCase):
   def setUp(self):
     super().setUp()
-    # TODO: add real data (append cases to existing tests)
-    deserialized_files = {PROPOSED: {}, SOLUTION: {}}
-    self.entity_connection_identification = EntityConnectionIdentification(
-        deserialized_files=deserialized_files).evaluate()
-
-  def testDirectlyAssignedAttributes(self):
-    self.assertEqual(
-        self.entity_connection_identification.correct_total_override, 0)
-    self.assertEqual(
-        self.entity_connection_identification.correct_ceiling_override, 0)
-    self.assertEqual(
-        self.entity_connection_identification.incorrect_total_override, 0)
-
-  def testInheritedCalculatedAttributes(self):
-    self.assertEqual(self.entity_connection_identification.correct_total(), 0)
-    self.assertEqual(self.entity_connection_identification.correct_ceiling(), 0)
-    self.assertEqual(self.entity_connection_identification.incorrect_total(), 0)
-
-  def testInheritedResultProperties(self):
-    # This is `None` by virtue of the ceiling being falsy.
-    self.assertEqual(self.entity_connection_identification.result_all, None)
+    featureful_file_path = (
+        'tests/samples/proposed/entity_connection_identification.yaml')
+    self.highest_score_argument = TestHelper.prepare_dimension_argument(
+        dimension=EntityConnectionIdentification,
+        proposed_path=featureful_file_path,
+        solution_path=featureful_file_path)
+
+    empty_file_path = 'tests/samples/empty.yaml'
+    self.none_score_argument = TestHelper.prepare_dimension_argument(
+        dimension=EntityConnectionIdentification,
+        proposed_path=empty_file_path,
+        solution_path=empty_file_path)
+
+    no_contains_file_path = 'tests/samples/proposed/entity_connection_identification_no_contains.yaml'  # pylint: disable=line-too-long
```

Comment:
why do we care of the specific case `no_contains` ?

Reply:
It's arbitrary — just what I eliminated to make fewer connections. Any naming suggestions?